locale/it/translation.js

```diff
@@ -0,0 +1,415 @@
+    shareContactURL: 'Condividi contatto URL', //Share Contact URL
+    setupYourContact: 'Imposta il tuo contatto', //Setup your Contact
+    receiveWithContact: 'Ricevi utilizzando un semplice contatto basato sul nome utente', //Receive using a simple username-based Contact
+    onlyShareContactPrivately: '<b>Solo</b> condividi i tuoi contatti con persone fidate (familiari, amici)', //<b>Only</b> share your Contact with trusted people (family, friends)
+
+    /* Context: The "Change to" is used in-app with one of the Three options below it, i.e: "Change to Contact" */
+    changeTo: 'Cambia in ', //Change to
+    contact: 'Contatto', //Contact
+    xpub: 'XPub', //XPub
+
+    addContactTitle: 'Aggiungi {strName} ai contatti', //Add {strName} to Contacts
+    addContactSubtext: 'Una volta aggiunto, sarai in grado di inviare transazioni a {strName} con il suo nome (digitando o facendo clic), niente più indirizzi, carino e facile.', //Once added you\'ll be able to send transactions to {strName} by their name (either typing, or clicking), no more addresses, nice \'n easy.
+    addContactWarning: 'Assicurati che questo sia il vero "{strName}", non accettare richieste di contatto da fonti sconosciute!', //Ensure that this is the real "{strName}", do not accept Contact requests from unknown sources!
+
+    editContactTitle: 'Cambia contatto "{strName}".', //Change "{strName}" Contact
+    newName: 'Nuovo Nome ', //New Name
+
+    removeContactTitle: 'Rimuovi{strName}?', //Remove {strName}?
+    removeContactSubtext: 'Sei sicuro di voler cancellare {strName} dai tuoi contatti?', //Are you sure you wish to remove {strName} from your Contacts?
+    removeContactNote: 'Puoi aggiungerlo di nuovo in qualsiasi momento in futuro.', //You can add them again any time in the future.
+
+    // Export
+    privateKey: 'Chiave Privata', //Private Key
+    viewPrivateKey: ' Visualizzare la chiave privata?', //View Private Key?
+    privateWarning1: 'Assicurati che nessuno possa vedere il tuo schermo.', //Make sure no one can see your screen.
+    privateWarning2: 'Chiunque abbia questa chiave può rubare i tuoi fondi.', //Anyone with this key can steal your funds.
+    viewKey: 'Viasualizza chiave', //View key
+
+    // PIVX Promos
+    // NOTE: for this below line, continue the sentence "PIVX Promos [...]", since 'PIVX Promos' is a brand, it is not translated.
+    pivxPromos:
+        "[...]è un sistema decentralizzato per codici regalo con valore PIV", // [...] is a decentralised system for gift codes worth PIV
+    // NOTE: on this line, if possible, leave 'PIVX Promos' untranslated
+    redeemInput: "Inserisci il tuo codice'PIVX Promos '", //Enter your 'PIVX Promos' code
+    createName: 'Nome della Promo(Facoltativo)', //Promo Name (Optional)
+    createAmount: 'Valore della promozione', //Promo Amount
```

Comment:
```suggestion
    createAmount: 'Valore del Promo', //Promo Amount
```
Give that "PIVX Promos" is not translated, I think it's reasonable to leave 'Promo' untranslated